src/Powercord/index.js

```diff
@@ -49,6 +50,10 @@ module.exports = class Powercord extends EventEmitter {
         SettingsManager.upload();
       }
     });
+
+    if (this.settings.get('hideToken')) {
+      require('powercord/webpack').getModule(['hideToken']).hideToken = () => void 0
```

Comment:
Shouldn't it be [ 'hideToken' ] because lintttttttttttttttttttt

Reply:
lol lint

Reply:
lol lint

Reply:
lol lint

Reply:
lol lint

Reply:
lol lint

Reply:
lol linted